### PR TITLE
refactor: Modularize CallTelemetry and tracking

### PR DESCRIPTION
**Pull request by ffflorian**

### CODE REVIEW

File: src/script/telemetry/calling/CallTelemetry.ts [hunk 1]
```diff
@@ -40,34 +54,11 @@ class CallTelemetry {
...
 
-  //##############################################################################
-  // Sessions
-  //##############################################################################
-
-  /**
-   * Force log last call session IDs.
-   * @returns {Object} Containing all the sessions
-   */
-  log_sessions() {
```
Comment (ffflorian) [inline src/script/telemetry/calling/CallTelemetry.ts:51]: This was not in use and I couldn't figure out how the `sessions` object was supposed to look like.

File: src/script/telemetry/calling/CallTelemetry.ts [hunk 2]
```diff
@@ -115,53 +99,37 @@ class CallTelemetry {
...
-          ].includes(eventName)
-            ? this.remote_version
-            : undefined,
-          started_as_video: videoTypes.includes(this.mediaType),
-          with_service: conversationEntity.hasService(),
-        },
-        trackingHelpers.getGuestAttributes(conversationEntity),
-        attributes
-      );
+      attributes = {
```
Comment (ffflorian) [inline src/script/telemetry/calling/CallTelemetry.ts:109]: This was done by TSLint / prettier

File: src/script/telemetry/calling/CallTelemetry.ts (hunk 1 above)
Comment (atomrc) [reply src/script/telemetry/calling/CallTelemetry.ts:51]: Could it be that QA was using it?

Comment (ffflorian) [reply src/script/telemetry/calling/CallTelemetry.ts:51]: Seems like not.

**Pull request merged**